### PR TITLE
[AIR][CUJ] E2E Pytorch training 

### PR DESCRIPTION
**Pull request by jiaodong**
## Why are these changes needed?

```
Resources requested: 0/16 CPUs, 0/1 GPUs, 0.0/71.7 GiB heap, 0.0/34.72 GiB objects (0.0/1.0 accelerator_type:M60)
Result logdir: /home/ray/ray_results/TorchTrainer_2022-07-15_16-33-33
Number of trials: 1/1 (1 TERMINATED)
+--------------------------+------------+---------------------+--------+------------------+----------------+--------------+---------------------+
| Trial name               | status     | loc                 |   iter |   total time (s) |   running_loss |   _timestamp |   _time_this_iter_s |
|--------------------------+------------+---------------------+--------+------------------+----------------+--------------+---------------------|
| TorchTrainer_8a60e_00000 | TERMINATED | 172.31.89.148:81011 |     10 |          104.834 |              0 |   1657928121 |             8.47623 |
+--------------------------+------------+---------------------+--------+------------------+----------------+--------------+---------------------+


2022-07-15 16:35:23,183 INFO tune.py:737 -- Total run time: 110.07 seconds (109.45 seconds for the tuning loop).
```

## Checks

- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

**Comment by jiaodong** [comment]
need to resolve conflict with master and https://github.com/ray-project/ray/pull/26614/files later.

**Review by ericl — APPROVED** [review]

File: release/air_tests/air_benchmarks/compute_gpu_1.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+cloud_id: {{env["ANYSCALE_CLOUD_ID"]}}
+region: us-west-2
+
+max_workers: 0
+
+head_node_type:
+    name: head_node
+    instance_type: g3.8xlarge
+
+worker_node_types:
```
Comment (amogkam) [inline release/air_tests/air_benchmarks/compute_gpu_1.yaml:10]: Remove if 0 workers?

**Review by amogkam — APPROVED** [review]

**Comment by amogkam** [comment]
Let’s make sure the release test is passing from the PR before merging this in

File: release/air_tests/air_benchmarks/compute_gpu_1.yaml (hunk 1 above)
Comment (jiaodong) [reply release/air_tests/air_benchmarks/compute_gpu_1.yaml]: i tried to request without this field but it's required 

```
HTTP response body: {"error":{"detail":[{"loc":["body","config","worker_node_types"],"msg":"field required","type":"value_error.missing"}],"body":{"name":"oss-release-tests_mWEC__compute__air_benchmark_pytorch_training_e2e_gpu_1x1__b495cab4ae6fcac8c3bde264dd55fe27a1a23f024d930625bfd5f0380dc8852f","project_id":"prj_mWECugke9RzMh79BZQqeykjN","config":{"cloud_id":"cld_401TPoxgB8MM6A0NNQauOV","region":"us-west-2","max_workers":0,"head_node_type":{"name":"head_node","instance_type":"g3.8xlarge"}}},

"message":"body.config.worker_node_types: field required"}}
```

**Pull request merged**